static/colors.css

```diff
@@ -0,0 +1,96 @@
+/* --------------------------------------------------------------------------
+ * DARK MODE ---------------------------------------------------------------- */
+
+:root {
+    --bg: black;
+    --bg-highlight: #161616;
```

Comment:
Why's it called `highlight`? :smile: It took me a while to figure out what this color is for. My expectation would be `codeblock`

Or perhaps `pre` but that's only clear if you know what the tag's called.

What do you think, could this be `--bg-codeblock`?

Reply:
I was trying to find something that encapsulates

1. the `pre`/code-blocks,
2. the alternating highlighted colors of tables, and
3. the `kbd`-tag background.

That's why I think `codeblock` or `pre` would be too specific.

We could do `bg-secondary` and rename the current `bg` to `bg-primary`. That would match the `text-` colors nicely and I think it's also at least somewhat conventional to have a "secondary background color" defined that should work for stuff that's on top of the "primary background color". What do you think?

Reply:
GitHub refers to these as "muted"

![image](https://github.com/user-attachments/assets/58182af8-02b2-4a03-816f-f37e7d0ea15f)

Reply:
Ah that's right, didn't realize how generic the color is.

> We could do `bg-secondary` and rename the current `bg` to `bg-primary`. That would match the `text-` colors nicely and I think it's also at least somewhat conventional to have a "secondary background color" defined that should work for stuff that's on top of the "primary background color". What do you think?

I like this idea the most :+1: